doc/cookbook/source/examples/classifier/multiclass_logisticregression.rst

```diff
@@ -0,0 +1,53 @@
+===============================
+Multi-class Logistic Regression
+===============================
+
+Logistic regression is a binary classifier, which assigns the :math:`d`-dimensional sample :math:`\mathbf{x}` to classes based on the probability :math:`p`:
+
+.. math::
+ 
+    \mathrm{z} = \alpha +  \mathbf{\beta x} = \alpha + \beta_{1}x_{1} + ... + \beta_{d}x_{d} = \mathbf{\theta}^{T}\mathbf{x}
+    
+.. math::
+
+    p = 1 / (1 + e^{-\mathrm{z}})
+    
+
+The loss function that needs to be minimized is:
+
+.. math::
+
+    {\min_{\mathbf{x}}}\sum_{l=1}^{k}\sum_{i=1}^{m}w_{il}log(1+exp(-y_{il}(\mathbf{x}_l^Ta_{il} + c_l))) + \lambda\left \| \mathbf{x} \right \|_{l_1/l_q}
+    
+where :math:`a_{il}` denotes the :math:`i`-th sample for the :math:`l`-th class, :math:`w_{il}` is the weight for :math:`a_{il}^T`, 
+:math:`y_{il}` is the response of :math:`a_{il}`, and :math:`c_l` is the intercept (scalar) for the :math:`l`-th class.
+    
```

Comment:
Sure :)
No worries if this takes a bit. It *is* complicated.
The math is not quite synchronised. Maybe this one helps? [link1](http://stats.stackexchange.com/questions/31714/logistic-regression-for-multiclass) [link2](http://blog.datumbox.com/machine-learning-tutorial-the-multinomial-logistic-regression-softmax-regression/)

You basically need to state two things
1) How the probability for a data x to be in class y is computed p(y_i=k | x_i)
2) The above will involve a parameter vector \theta. You need to state the loss that is minimised over \theta (not over the data x itself btw). 
3) The loss will be regularised, which has different implications. The slep manual I posted contained the way it is done here.

Currently there are some problems.
 * min is over x, which doesnt make sense, it should be over \theta
 * the p in first line is only for one class

Just combine all pieces of information and then we will have it running :)